### PR TITLE
Change json mapping to reflect backend change

### PR DESCRIPTION
**Pull request by JinghuaWang**
The backend has changed JSON keys to match the JSON naming convention.
The frontend will now change the JSON mapping to correctly serialize data from the backend event API

### CODE REVIEW

File: frontend/lib/json/event.dart [hunk 1]
```diff
@@ -1,54 +1,27 @@
-// class EventInfo {
```
Comment (JinghuaWang) [inline frontend/lib/json/event.dart:1]: Remove unused code

File: frontend/lib/json/event.dart [hunk 2]
```diff
@@ -83,16 +56,16 @@ class EventInfo {
       eventName: json['event_name'],
       description: json['description'],
       hostId: json['host_id'],
-      isPublic: json['isPublic'],
```
Comment (JinghuaWang) [inline frontend/lib/json/event.dart:86]: Change the keys to proper JSON names

**Review by JinghuaWang — COMMENTED** [review]
Send the pull request to backend developer for code review

File: frontend/lib/json/event.dart (hunk 1 above)
Comment (wzhang88) [reply frontend/lib/json/event.dart:1]: Looks good to me. Frontend and backend now are consistent

File: frontend/lib/json/event.dart (hunk 2 above)
Comment (wzhang88) [reply frontend/lib/json/event.dart:86]: great! should be able to retrieve data correctly

**Review by wzhang88 — APPROVED** [review]
Should be good now

**Pull request merged**